docs/user/image_mode.md

```
@@ -1,235 +1,168 @@
+Linux kernel, which is used to boot.
 
 MicroShift build and deployment procedures can utilize bootable containers to
 benefit from this technology.
 
-This document demonstrates how to create a bootable container image, store this
-image in a remote registry and use it for installing a new RHEL operating system.
-
 > See [Image mode for Red Hat Enterprise Linux](https://developers.redhat.com/products/rhel-image-mode/overview)
-for more information.
+> for more information.
+
```

markdownlint-cli2 (0.23.2)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)